notification/src/config/config.js

```diff
@@ -23,6 +23,13 @@ function getCTPEnvCredentials () {
   }
 }
 
+function getAdyenCredentials () {
+  return {
+    secretHMACKey: process.env.ADYEN_SECRET_HMAC_KEY,
+    disableHMACSignature: process.env.DISABLE_HMAC_SIGNETURE || false
```

Comment:
```suggestion
    disableHMACSignature: process.env.DISABLE_HMAC_SIGNATURE || false
```

Reply:
I find it easier to use positive verbs (enable) instead of negative (disable). It's easier for understanding especially at the if conditions (https://github.com/commercetools/commercetools-adyen-integration/pull/273/files#diff-a33fb40063ad0e73a4580bec72d1bec5R64 - it takes some time here to know which value is evaluated to true).

`ENABLE_HMAC_SIGNATURE || true`

Reply:
The values from `process.env` are evaluated as String, so I think you need to parse 'false' and 'true' to Boolean.

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/273/commits/6ec9853e1874ec905f07ad6f67e3457883764a06